doc/plugins/presale-saml.rst

```diff
@@ -0,0 +1,399 @@
+.. highlight:: ini
+
+Presale SAML Authentication
+===========================
+
+The Presale SAML Authentication plugin is an advanced plugin, which most event
+organizers will not need to use. However, for the select few  who do require
+strong customer authentication that cannot be covered by the built-in customer
+account functionality, this plugin allows pretix to connect to a SAML IdP and
+perform authentication and retrieval of user information.
+
+Usage of the plugin is governed by two separate sets of settings: The plugin
+installation, the Service Provider (SP) configuration and the event
+configuration.
+
+Plugin installation and initial configuration
+---------------------------------------------
+
+.. note:: If you are a customer of our hosted `pretix.eu`_ offering, you can
+          skip this section.
+
+The plugin is installed as any other plugin in the pretix eco-system. As a
+pretix system administrator, please follow the instructions in the the
+:ref:`Administrator documentation <admindocs>`.
+
+Once installed, you will need to assess, if you want (or need) your pretix
+instance to be a single SP for all organizers and events or if every event
+organizer has to provide their own SP.
+
+Take the example of a university which runs pretix under an pretix Enterprise
+agreement. Since they only provide ticketing services to themselves (every
+organizer is still just a different department of the same university), a
+single SP should be enough.
+
+On the other hand, a reseller such as `pretix.eu`_ who services a multitude
+of clients would not work that way. Here, every organizer is a separate
+legal entity and as such will also need to provide their own SP configuration:
+Company A will expect their SP to reflect their company - and not a generalized
+"pretix SP".
+
+Once you have decided on the mode of operation, the :ref:`Configuration file
+<config>` needs to be extended to reflect your choice.
+
+Example::
+
+    [presale-saml]
+    level=global
+
+``level``
+    ``global`` to use only a single, system-wide SP, ``organizer`` for multiple
+    SPs, configured on the organizer-level. Defaults to ``organizer``.
+
+Service Provider configuration
+------------------------------
+
+Global Level
+^^^^^^^^^^^^
+
+.. note:: If you are a customer of our hosted `pretix.eu`_ offering, you can
+          skip this section and follow the instructions on the upcoming
+          Organizer Level settings.
+
+As a user with administrative privileges, please activate them by clicking the
+``Admin Mode`` button in the top right hand corner.
+
+You should now see a new menu-item titled ``SAML`` appear.
+
+Organizer Level
+^^^^^^^^^^^^^^^
+
+Navigate to the organizer settings in the pretix backend. In the navigation
+bar, you will find a menu-item titled ``SAML`` if your user has the ``Can
+change organizer settings`` permission.
```

Comment:
I don't feel that this is the correct markup for non-code, I'd either use italics or just quotation marks